IOTATransferRestRequestTemplate.py

```diff
@@ -82,12 +77,14 @@ def output_seed(seed):
 
 # sample dummy api request, will return dictionary of request object and UUID of the requst
 
-def create_request(url="https://api.github.com/users/sigmoidfreud/repos", headers={}):
-    return {"request": requests.get(url, headers), "request-id": uuid.uuid5()}
+def create_request(url="https://api.github.com/users/sigmoidfreud/repos", headers=None):
+    # type: (Text, Optional[dict]) -> dict
+    return {"request": requests.get(url, headers or {}), "request-id": uuid.uuid4()}
```

Comment:
I think you meant to use `uuid.uuid4` here instead of `uuid.uuid5`.

- `uuid4` generates a completely random value.
- `uuid5` requires a namespace argument.